sdk/client-core/core/src/main/java/io/clientcore/core/util/binarydata/BinaryData.java

```diff
@@ -115,6 +115,8 @@
  * @see <a href="https://aka.ms/azsdk/java/docs/serialization" target="_blank">More about serialization</a>
  */
 public abstract class BinaryData implements Closeable {
+    public static final BinaryData EMPTY = BinaryData.fromBytes(new byte[0]);
```

Comment:
This needs documentation and do we want to make this private and only allow access to EMPTY through `BinaryData.empty()`

Reply:
We should use the same pattern as RequestOptions.NONE, Context.NONE.